### PR TITLE
Add RayOnSpark examples

### PR DESCRIPTION
**Pull request by HzhElena**
a dummy request

### CODE REVIEW

**Comment by shane-huang** [comment]
Create a folder "pyzoo/zoo/examples/ray" and put your .keep file into "ray" , instead of into "examples". Just push commits to your own branch you will see changes refreshed in this pr. Later when you finished enabling pong example, put the example into a folder like "pyzoo/zoo/examples/ray/pong". I changed the title of this PR since it is very confusing.

**Comment by HzhElena** [comment]
PR Validation Succeed

File: pyzoo/zoo/examples/run-example-tests.sh [hunk 1]
```diff
@@ -364,6 +364,59 @@ ${SPARK_HOME}/bin/spark-submit \
...
+then
+    clear_up
+    echo "sync_parameter_server failed"
+    exit $exit_status
+fi
+now=$(date "+%s")
+time10=$((now-start))
+
+
+echo "start example async_parameter_server"
```
Comment (zhichao-li) [inline pyzoo/zoo/examples/run-example-tests.sh]: Wrap these duplicate code into function? 
i.e
```
execute_test () {
  xxx
}
execute_test   async_parameter_server "python ${ANALYTICS_ZOO_ROOT}/pyzoo/zoo/examples/ray/parameter_server/sync_parameter_server.py"
```

File: pyzoo/zoo/examples/run-example-tests.sh [hunk 2]
```diff
@@ -364,6 +364,32 @@ ${SPARK_HOME}/bin/spark-submit \
 now=$(date "+%s")
 time8=$((now-start))
 
+execute_ray_test () {
```
Comment (zhichao-li) [inline pyzoo/zoo/examples/run-example-tests.sh]: cc @glorysdj  I suppose this test file is dedicated to yarn-client mode?

File: pyzoo/zoo/examples/ray/.keep [hunk 3]
```diff
@@ -0,0 +1 @@
+first pr
```
Comment (zhichao-li) [inline pyzoo/zoo/examples/ray/.keep]: remove this file

File: pyzoo/zoo/examples/run-example-tests-pip.sh [hunk 4]
```diff
@@ -375,6 +376,38 @@ now=$(date "+%s")
 time9=$((now-start))
 echo "qaranker time used:$time9 seconds"
 
+execute_ray_test () {
+    echo "start example $1"
+    start=$(date "+%s")
+    python $2
```
Comment (zhichao-li) [inline pyzoo/zoo/examples/run-example-tests-pip.sh]: Move python to $2. The idea is the test function should be able to accept any bash command not only python.

File: pyzoo/zoo/examples/run-example-tests-pip.sh [hunk 5]
```diff
@@ -375,6 +376,38 @@ now=$(date "+%s")
...
+    exit_status=$?
+    if [ $exit_status -ne 0 ];
+    then
+        clear_up
+        echo "$1 failed"
+        exit $exit_status
+    fi
+    now=$(date "+%s")
+    return $((now-start))
```
Comment (zhichao-li) [inline pyzoo/zoo/examples/run-example-tests-pip.sh]: There is a limitation for using stdout as the return value as it's dedicated to execution status with range.
Change to this:
```
my_function () {  
  elapse_time=$((now-start))
} 
 
time9=${elapse_time}
```

**Comment by shane-huang** [comment]
This passed pr validation now?

**Comment by zhichao-li** [comment]
I think it's still pending on the new image of the Jenkins environment.

**Pull request merged**